### DISCUSSION TITLE
Why are my Images being deleted out of /images ??

### DISCUSSION THREAD

**Discussion by c1972** [thorsten/phpMyFAQ#3029] [Q&A]
I uploaded some images to **/images** via FTP.

I then linked the images in posts in the phpMyFAQ editor. Everything worked great.

Until, after an hour or two, the images are suddenly **DISAPPEARED** from the folder! 👀

What is the system doing? Why is this happening?

Should I save my images in **/assets/themes/mytemplate/img/** instead? ❓❓❓

Seems I have to change a lot of URLs now.  😡

**Comment by thorsten** [comment]
There is no functionality to delete images from the images/ folder.

**Comment by thorsten** [comment]
> Should I save my images in **/assets/themes/mytemplate/img/** instead? ❓❓❓

no, because then you can't access them with the editor
